conans/test/functional/editable/link_remove_test.py

```diff
@@ -21,8 +20,8 @@ def setUp(self):
 
         self.t = TestClient()
         self.t.save(files={'conanfile.py': self.conanfile,
-                           CONAN_PACKAGE_LAYOUT_FILE: "", })
-        self.t.run('link . {}'.format(self.ref))
+                           "mylayout": "", })
+        self.t.run('link . {} -l=mylayout'.format(self.ref))
```

Comment:
Check the output: `linked to path ....`

Reply:
Not the correct place, a setUp() for a remove test suite.

The "linked to path" functionality is tested in a "link" test.

Reply:
Totally agree!